src/Renci.SshNet/ConnectionInfo.cs

```diff
@@ -174,6 +176,12 @@ public class ConnectionInfo : IConnectionInfoInternal
         /// </value>
         public int RetryAttempts { get; set; }
 
+        /// <summary>
+        /// Gets or sets the local IP address to bind.
+        /// If not null, bind local socket to given address (use this if you have several interfaces and want to be seen as connecting from a particular one).
+        /// </summary>
+        public IPAddress LocalIpAddress { get; set; }
```

Comment:
Why not `IPEndpoint LocalEndpoint` ? This way the user can optionally specify a local port as well, which might be needed for some strict firewalls.